app.js

```diff
@@ -303,4 +312,10 @@ app.use((err, req, res, next) => {
 	});
 });
 
+process.on('unhandledRejection', (err, res, req, next) => {
```

Comment:
Nice. It would be good to add 
`process.on('uncaughtException', (err, origin) => {
`

In both cases it is advised to close requested resources while handling the exception. Otherwise there is a chance that for example the HTTP request remains open.

https://stackoverflow.com/questions/5999373/how-do-i-prevent-node-js-from-crashing-try-catch-doesnt-work#comment9952914_5999381